l1-contracts/test/Decoder.t.sol

```diff
@@ -143,8 +147,8 @@ contract DecoderTest is Test {
 
     // Note: First 32 bytes are 0 because those correspond to the hash of previous iteration and there was no previous
     //       iteration.
-    // keccak256(hex"0000000000000000000000000000000000000000000000000000000000000000aafdc7aa93e78a70");
-    bytes32 referenceLogsHash = 0x9a60e4b675115fcedf18aa62d374d0616192a1147bd81716f1c482f48363fc74;
+    // bytes32(uint256(keccak256(hex"0000000000000000000000000000000000000000000000000000000000000000aafdc7aa93e78a70")) % P);
+    bytes32 referenceLogsHash = 0x0933f95dd17c7f51b627d93f4ef0c749e8f6e83b0eabc563261ea238b363fc71;
```

Comment:
Why is this using `keccak256` and is there a reason the value is hardcoded not using the commented  line?

Reply:
Addressed in [745cf77](https://github.com/AztecProtocol/aztec-packages/pull/615/commits/745cf77eb76c11e4e14c8759479ab5d36375170f)